### DISCUSSION TITLE
Clarify possible usage of html parameter

### DISCUSSION THREAD

**Discussion by pavelsr** [allejo/jekyll-anchor-headings#15] [Q&A] [answered]
Vladimir, could you please explain me why [such jekyll-anchor-headings addition](https://github.com/pavelsr/jekyll-toc-html-test/commit/12cf32530e04dba0c698c80e8d18332fd083192f) is not working? I will owe you a beer :)

P.S. [allejo/jekyll-toc](https://github.com/allejo/jekyll-toc) is working same way, but [jekyll-anchor-headings](https://github.com/allejo/jekyll-anchor-headings) doesn't. Seems like difference in [html parameter](https://github.com/allejo/jekyll-anchor-headings#parameters) usage.

P.P.S. If you have [similar](https://allejo.io/blog/a-jekyll-toc-in-liquid-only/) blog post about jekyll-anchor-headings I will appreciate this.

**Comment by allejo** [comment, marked as answer]
> Vladimir, could you please explain me why [such jekyll-anchor-headings addition](https://github.com/pavelsr/jekyll-toc-html-test/commit/12cf32530e04dba0c698c80e8d18332fd083192f) is not working? I will owe you a beer :)
> 
> P.S. [allejo/jekyll-toc](https://github.com/allejo/jekyll-toc) is working same way, but [jekyll-anchor-headings](https://github.com/allejo/jekyll-anchor-headings) doesn't. Seems like difference in [html parameter](https://github.com/allejo/jekyll-anchor-headings#parameters) usage.

Usage is the same. Check the source code of the generated page, you'll see there are `<a></a>` tags in there :)

Just add an `anchorBody` parameter to the include. It's left empty so people can add icons via CSS and `content: ''` instead of having them override the `anchorBody`.

```liquid
{% include anchor_headings.html html=foo anchorBody="#" %}
```

> P.P.S. If you have [similar](https://allejo.io/blog/a-jekyll-toc-in-liquid-only/) blog post about jekyll-anchor-headings I will appreciate this.

Sure! https://allejo.io/blog/jekyll-heading-anchors-without-javascript/

**Comment by pavelsr** [comment]
Thanks! `anchorBody="#"` did the trick.

[Bought](https://www.buymeacoffee.com/allejo/c/100457) you a coffee 😉

**Comment by allejo** [comment]
> [Bought](https://www.buymeacoffee.com/allejo/c/100457) you a coffee 😉

Thank you so much! :smile: :heart:
